_unittests/test_pkg-config.cmake

```diff
@@ -0,0 +1,34 @@
+include(cmake/pkg-config.cmake)
+
+cmake_policy(SET CMP0054 NEW)
+#cmake -P
+macro(EXPECT_STREQUAL _lhs _rhs)
+  if(NOT "${_lhs}" STREQUAL ${_rhs})
+    message(SEND_ERROR "EXPECT_STREQUAL failed: \"${_lhs}\" != \"${_rhs}\"")
+  endif()
+endmacro()
+
+macro(UNSET_TEST_VARS)
+  UNSET(PKG_LIB_NAME)
+  UNSET(PKG_PREFIX)
+  UNSET(PKG_CONFIG_STRING_NOSPACE)
+endmacro()
+
+UNSET_TEST_VARS()
+_PARSE_PKG_CONFIG_STRING("my-package > 0.4" PKG_LIB_NAME PKG_PREFIX PKG_CONFIG_STRING_NOSPACE)
+EXPECT_STREQUAL("my-package" "${PKG_LIB_NAME}")
+EXPECT_STREQUAL("MY_PACKAGE" "${PKG_PREFIX}")
+EXPECT_STREQUAL("my-package>0.4" "${PKG_CONFIG_STRING_NOSPACE}")
+
+UNSET_TEST_VARS()
+_PARSE_PKG_CONFIG_STRING("my-package >= 0.4" PKG_LIB_NAME PKG_PREFIX PKG_CONFIG_STRING_NOSPACE)
+EXPECT_STREQUAL("my-package" "${PKG_LIB_NAME}")
+EXPECT_STREQUAL("MY_PACKAGE" "${PKG_PREFIX}")
+EXPECT_STREQUAL("my-package>=0.4" "${PKG_CONFIG_STRING_NOSPACE}")
+
+UNSET_TEST_VARS()
+_PARSE_PKG_CONFIG_STRING("my-package" PKG_LIB_NAME PKG_PREFIX PKG_CONFIG_STRING_NOSPACE)
+EXPECT_STREQUAL("my-package" "${PKG_LIB_NAME}")
+EXPECT_STREQUAL("MY_PACKAGE" "${PKG_PREFIX}")
+EXPECT_STREQUAL("my-package" "${PKG_CONFIG_STRING_NOSPACE}")
+
```

Comment:
Can you test the version without spaces too ?

Reply:
Thanks.

Reply:
I planed to *not* support the version without space.
It was never supported by jrl-cmakemodules AFAIK, and it seems better to stick to pkg-config's behavior than to support FindPkgConfig.cmake's one.